packages/amplify-cli-core/API.md

```diff
@@ -376,6 +402,9 @@ export enum CFNTemplateFormat {
     YAML = "yaml"
 }
 
+// @public (undocumented)
+export const checkIfAuthExists: () => boolean;
```

Comment:
if we're in core should we have `checkIfCategoryExists(categoryName:string)` ?

Reply:
I propose we namespace all the amplify toolkit extensions when we export them from `-core`. So instead of doing a flat re-export like we're doing, these would be exported as `toolkitExtensions.checkIfAuthExists`. This will help us keep `core`'s surface area at least a bit better under control. It will also provide slightly clearer namespacing to hint at these methods' reason for being.

Reply:
I think it's a good step forward for now but ultimately we shouldn't be able to full text search core and find `auth` there. Let's do what you propose for now and log follow up.